api/util/api/api_test.go

```diff
@@ -0,0 +1,108 @@
+package util_test
+
+import (
+	"errors"
+	"github.com/globocom/glbgelf"
+	apiContext "github.com/globocom/huskyCI/api/context"
+	apiUtil "github.com/globocom/huskyCI/api/util/api"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+type CheckHuskyData struct {
+	configApi             *apiContext.APIConfig
+	envVarsError          error
+	dockerHostsError      error
+	mongoDBError          error
+	eachSecurityTestError error
+	expectedError         error
+}
+
+var checkHuskyTests = []CheckHuskyData{
+	// Scenario #1: checkEnvVars returns an error
+	{&apiContext.APIConfig{}, errors.New("Falha ao verificar variáveis de ambiente"), nil, nil, nil, errors.New("Falha ao verificar variáveis de ambiente")},
```

Comment:
Would you mind using English in these error messages? 🙃

Reply:
Mó!

Reply:
Error messages translated to English [here](https://github.com/globocom/huskyCI/pull/311/commits/1406d408c4c6bd59bf24b4db5f2edf79774314dc).